test-files/cases/differential/unifying/generic-geometry/generic-3894-1/Expected.osm

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <osm version="0.6" generator="hootenanny" srs="+epsg:4326">
     <bounds minlat="13.2761767" minlon="12.5703467" maxlat="13.3438904" maxlon="12.6285334"/>
```

Comment:
This test output got ever so slightly worse with these changes. There are two small linear features that arguably shouldn't be in the output diff but now are. In both cases, they are unclosed linear barriers. I'm ok with it. If someone wants to rerun with completely match removal, then they will get slightly better output.